manifests/profile/prometheus/exporter/node.pp

```diff
@@ -56,10 +57,33 @@
 
   file { '/etc/systemd/system/prometheus-node-exporter.service':
     content => template('nebula/profile/prometheus/exporter/node/systemd.ini.erb'),
-    notify  => Service['prometheus-node-exporter'],
+    notify  => [Service['prometheus-node-exporter'], Exec['reload systemctl daemons for prometheus']],
```

Comment:
I am a bit on the fence on this. On one hand, I would rather see this as a generic systemctl reload not specific to prometheus, so that additional notifications in unrelated profiles do not result in more than one reload. On the other, this follows the pattern set in our `docker.pp` profile. This is acceptable in its current form.

Reply:
Subscribers now exist, which kind of addresses this.